java/ql/src/Security/CWE/CWE-200/AndroidWebViewSettingsFileAccess.qhelp

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE qhelp PUBLIC
+ "-//Semmle//qhelp//EN"
+ "qhelp.dtd">
+<qhelp>
+  <overview>
+    <p>
+      File access in an Android WebView can expose the device's file system to
+      the JavaScript running in the WebView. If there are vulnerabilities in the
+      JavaScript, file access may allow an attacker to access or steal the
+      user's data.
+    </p>
+  </overview>
+
+  <recommendation>
+    <p>When possible, you should disallow file access by setting the following settings to <code>false</code>:</p>
```

Comment:
I would mention this is `false` by default, so leaving it untouched would also suffice to prevent the issue. Also, we should probably recommend the use of [`AssetLoader`](https://developer.android.com/reference/androidx/webkit/WebViewAssetLoader) instead as well (see the note in https://developer.android.com/reference/android/webkit/WebSettings#setAllowFileAccess(boolean)).